.github/workflows/build-domjudge-container-PR.yml

```diff
@@ -73,7 +73,7 @@ jobs:
         run: |
           for IMG in domserver judgehost default-judgehost-chroot; do
             echo "::group::$IMG"
-            docker list
+            docker ps
```

Comment:
There is an earlier failure:

```
#22 [stage-1 9/9] RUN chmod 700 /configure.sh && /configure.sh && rm -f /configure.sh
#22 0.163 useradd: user 'domjudge' already exists
#22 ERROR: process "/bin/sh -c chmod 700 /configure.sh && /configure.sh && rm -f /configure.sh" did not complete successfully: exit code: 9
------
 > [stage-1 9/9] RUN chmod 700 /configure.sh && /configure.sh && rm -f /configure.sh:
0.163 useradd: user 'domjudge' already exists
------
Dockerfile:82
--------------------
  80 |     # Run customizations
  81 |     COPY domserver/configure.sh /configure.sh
  82 | >>> RUN chmod 700 /configure.sh && /configure.sh && rm -f /configure.sh
  83 |     
  84 |     # Expose HTTP port
--------------------
ERROR: failed to solve: process "/bin/sh -c chmod 700 /configure.sh && /configure.sh && rm -f /configure.sh" did not complete successfully: exit code: 9
+ echo [ok] Done building Docker image for domserver
```

It's being ignored because of an issue that I'm fixing in #197.